### PR TITLE
Adds quantified associations on swagger

### PR DESCRIPTION
**Pull request by fitn**

### CODE REVIEW

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 1]
```diff
@@ -46,6 +46,34 @@ properties:
             data:
               type: object
               description: Product value
+  quantified_associations:
+    type: object
+    description: Several quantified associations related to products and/or product models, grouped by quantified association types (only available in SaaS and sincce the v5.0)
```
Comment (NolwennP) [inline content/swagger/resources/product_models/definitions/product_model.yaml:51]:
```suggestion
    description: Several quantified associations related to products and/or product models, grouped by quantified association types (only available in Serenity)
```

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 2]
```diff
@@ -117,6 +145,30 @@ example: {
         "data": "Biker jacket"
       }
     ]
+  },,
```
Comment (NolwennP) [inline content/swagger/resources/product_models/definitions/product_model.yaml:148]:
```suggestion
  },
```

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 3]
```diff
@@ -117,6 +145,30 @@ example: {
...
+    "PACK": {
+      "products": [
+      {
+        "identifier": "top",
+        "quantity": 2
+      },
+      {
+        "identifier": "cap",
+        "quantity": 1
+      }
```
Comment (NolwennP) [inline content/swagger/resources/product_models/definitions/product_model.yaml:159]: Indentation missing

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 4]
```diff
@@ -117,6 +145,30 @@ example: {
...
+      ],
+      "product_models": [
+      {
+        "code": "model-biker-jacket-leather",
+        "quantity": 2
+      }
```
Comment (NolwennP) [inline content/swagger/resources/product_models/definitions/product_model.yaml:165]: Missing indentation

File: content/swagger/resources/product_models/routes/product_models.yaml [hunk 5]
```diff
@@ -96,6 +96,30 @@ get:
                     "data": "I like shoes!"
                   }
                 ]
+              },,
```
Comment (NolwennP) [inline content/swagger/resources/product_models/routes/product_models.yaml:99]:
```suggestion
              },
```

File: content/swagger/resources/product_models/routes/product_models.yaml [hunk 6]
```diff
@@ -96,6 +96,30 @@ get:
...
+                "PACK": {
+                  "products": [
+                  {
+                    "identifier": "cap",
+                    "quantity": 2
+                  },
+                  {
+                    "identifier": "shoes",
+                    "quantity": 1
+                  }
```
Comment (NolwennP) [inline content/swagger/resources/product_models/routes/product_models.yaml:110]: Missing indentation

File: content/swagger/resources/product_models/routes/product_models.yaml [hunk 7]
```diff
@@ -96,6 +96,30 @@ get:
...
+                  ],
+                  "product_models": [
+                  {
+                    "identifier": "model-biker-jacket-leather",
+                    "quantity": 2
+                  }
```
Comment (NolwennP) [inline content/swagger/resources/product_models/routes/product_models.yaml:116]: Missing indentation

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 8]
```diff
@@ -117,7 +145,31 @@ example: {
         "data": "Biker jacket"
       }
     ]
-  },
+  },,
+  "quantified_associations": {
+    "PACK": {
+      "products": [
+      {
```
Comment (juliensnz) [inline content/swagger/resources/product_models/definitions/product_model.yaml:152]: this should be indented 2 spaces more

File: content/swagger/resources/product_models/routes/product_models.yaml [hunk 9]
```diff
@@ -96,7 +96,31 @@ get:
                     "data": "I like shoes!"
                   }
                 ]
-              },
+              },,
+              "quantified_associations": {
+                "PACK": {
+                  "products": [
+                  {
```
Comment (juliensnz) [inline content/swagger/resources/product_models/routes/product_models.yaml:103]: same here

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 10]
```diff
@@ -46,6 +46,34 @@ properties:
...
+          products:
+            type: array
+            description: Array of objects containing product identifiers and quantities with which the product is in relation
+            items:
+              type: object
+              properties:
+                identifier:
+                  type: string
+                quantity:
+                  type: string
```
Comment (StevenVAIDIE) [inline content/swagger/resources/product_models/definitions/product_model.yaml:66]: @fitn the quantity should be integer or string ? In example it's an integer

**Comment by NolwennP** [comment]
Hello @fitn!
Why is this PR still open? It is weird as it seems that those changes have already been merged in master. Should we close it?

**Comment by StevenVAIDIE** [comment]
Those changes have been merged into master => https://github.com/akeneo/pim-api-docs/pull/387

**Pull request closed without merging**